packages/schematics/angular/e2e/schema.json

```diff
@@ -26,6 +26,10 @@
     "relatedAppName": {
       "description": "The name of the app being tested.",
       "type": "string"
+    },
+    "prefix": {
+      "description": "The prefix to apply.",
+      "type": "string"
```

Comment:
Should probably have a default or some way to pickup the value from an existing project on e2e so people can do `ng generate e2e` without a prefix.

Reply:
@hansl I added a default (`app` as we do everywhere for prefix. Also added `format: html-selector` and `alias: p` to be coherent.